src/redux/books/books.js

```diff
@@ -0,0 +1,42 @@
+import { apiAddBook, apiGetBooks, apiRemoveBook } from '../../modules/api';
+
+const ADD = 'bookstore/Book/ADD';
+const GET = 'bookstore/Book/GET';
+const REMOVE = 'bookstore/Book/REMOVE';
+
+export const addBook = (payload) => async (dispatch) => {
+  const book = { ...payload, item_id: payload.id, category: 'category' };
+  await apiAddBook(book);
+  dispatch({ type: ADD, book });
+};
+
```

Comment:
![Capture](https://user-images.githubusercontent.com/93039359/206868762-eb27958f-00cb-4fe9-82e5-56c6f9d9b64a.PNG)

- [ ] It's great that you have a license file in your project but it looks like a template 🤔 Kindly add your **name**, **current year** and **app-type** 😊 to make your project licensed to you 👍